mhlib/app.go

```diff
@@ -92,6 +93,11 @@ func NewApp(logger *logrus.Entry, appConfig AppConfig, mhConfig MHConfig) (*App,
 		appConfig.Key = fmt.Sprintf(".%s", strcase.LowerCamelCase(id))
 	}
 
+	// Set ".Namespace" to "default" if not defined
+	if appConfig.Namespace == "" {
+		appConfig.Namespace = "default"
```

Comment:
One small nit - my understanding of helm charts is that they get installed to the current namespace based on `kubeconfig` context (probably most of them, so far in a month I haven't found any doing it differently but I can imagine it could happen). Wouldn't this force the `default` namespace if `appConfig.Namespace` is undefined?

Reply:
You are probably right. Since all of our kubeconfigs set default right now, it's probably not an urgent thing to fix.  Could you file an issue for that?  Current context is easily available to us, if I recall the code correctly. Should be an easy fix.  Thanks!